frontend/src/bundles/common/components/header/header.tsx

```diff
@@ -16,8 +16,7 @@ const Header: React.FC<Properties> = ({ left, center, right }) => {
             left="0"
             width="full"
             backgroundColor="background.900"
-            boxShadow="md"
-            zIndex="100"
+            zIndex="1000"
```

Comment:
Does zIndex="100" not work?
Because in [this pr](https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/109/files) we changed it from 1000 to 100 :)